Clean up file comments and update to Docsy v0.1.0

NatasaKralj opened the pull request:
I changed comment formatting in html partials from html comments to go template comments.
I adjusted comments and contents in SCSS files to match Docsy v0.1.0.
I added a fix for the height of headers, to allow for selection of items above them.
I adjusted layouts files to match Docsy v0.1.0 templates.

NatasaKralj committed:
Change comment formatting for go templating

NatasaKralj committed:
Change comment formatting for go templating

NatasaKralj committed:
Adjusted comments and contents of SCSS files to Docsy v0.1.0

NatasaKralj committed:
Add fix for header selection height

NatasaKralj committed:
Update partials to Docsy v0.1.0

NatasaKralj committed:
Update partials to Docsy v0.1.0

NatasaKralj committed:
Add comments for cleanup

MarkvanMents commented on layouts/partials/navbar.html:
```diff
@@ -1,7 +1,7 @@
-<!-- {{ $cover := .HasShortcode "blocks/cover" }}
+{{/* {{ $cover := and (.HasShortcode "blocks/cover") (not .Site.Params.ui.navbar_translucent_over_cover_disable) }}
```
Should we have a comment here?
Do we actually need this, or have we commented out every call to navbar and therefore we don't have to overwrite the Docsy version at all?

NatasaKralj replied:
If we don't comment it out, we have to add 3 more files from Docsy into layouts to comment out the call to navbar.

NatasaKralj replied:
Added a comment to the top of the file for more information.

MarkvanMents commented on layouts/partials/print/render.html:
```diff
@@ -54,18 +54,18 @@
 {{ partial "print/page-heading.html" . }}
 
-<!-- MvM: Remove ToC from the printed section pages
+{{/* MvM: Remove ToC from the printed section pages
```
**Not related to comment style!**
~This doesn't seem to have worked - need to rethink which comments need to be in HTML comments and which in GoTemplate comments.~
![image](https://user-images.githubusercontent.com/35492184/160642571-214873de-2c49-4be0-8004-2c2179ae76d2.png)

MarkvanMents replied:
Oh no - wasn't that? Perhaps I'm misremembering. It is the other way around on the current site 😅 so that wasn't quite right either.
But doesn't look like it is a comment issue, so I take that back.
![image](https://user-images.githubusercontent.com/35492184/160645606-ac0962f5-7380-49c4-aea9-7f25879fcc1d.png)

MarkvanMents reviewed (dismissed):
Seems to be a different issue. Unrelated to comment style. Ignore comment below.

~Need to review whether we are doing comments right. See comments below, but one of the changes certainly hasn't worked.~
~I think that the {{ /* comments work in documents, but in partials/shortcodes the templater doesn't recognize them? Or perhaps having {{ inside {{ messes things up.~

~We should probably go back to using <!-- comments in most places, and then review what gets built to see if we need to change anything?~

MarkvanMents commented on layouts/docs/baseof.print.html:
```diff
@@ -3,19 +3,15 @@
     <div class="container-fluid td-outer">
       <div class="td-main">
         <div class="row flex-xl-nowrap stretch-main">
-          <div class="col-12 col-md-3 col-xl-2 td-sidebar d-print-none">
```
These are the lines which were making the print gutter appear to the left rather than the right!

MarkvanMents commented on layouts/docs/baseof.print.html:
```diff
@@ -3,19 +3,15 @@
-          </div>
-          <div class="d-none d-xl-block col-xl-2 td-toc d-print-none">
-          </div>
           <main class="col-12 col-md-9 col-xl-8 pl-md-5" role="main">
```
This displays the print version on the whole screen without empty columns on either side
```suggestion
          <main class="col-12 col-md-12 col-xl-12 pl-md-5" role="main">
```
Can you try this out and see if it looks OK for you.

NatasaKralj replied:
Fixed with `<main class="col-12 pl-md-5" role="main">`

MarkvanMents reviewed (commented):
Found the strange behaviour with the print partial

NatasaKralj committed:
Remove any menu from print version

NatasaKralj committed:
Add comment to partial

NatasaKralj committed:
Update partials to Docsy v0.1.0
Change edit button to view

NatasaKralj commented:
I fixed the last 2 partials to be updated to Docsy v0.1.0.
I also changed the Edit button property to take the user to the file blob on GH, but the edit page directly.

NatasaKralj committed:
Remove old comment

NatasaKralj commented:
Approved via slack, merging
